config/env.ini

```diff
@@ -117,8 +117,6 @@ SPC_CMD_PREFIX_PHP_BUILDCONF="./buildconf --force"
 SPC_CMD_PREFIX_PHP_CONFIGURE="./configure --prefix= --with-valgrind=no --enable-shared=no --enable-static=yes --disable-all --disable-cgi --disable-phpdbg"
 ; make command
 SPC_CMD_PREFIX_PHP_MAKE="make -j${CPU_COUNT}"
-; embed type for php, static or shared
-SPC_CMD_VAR_PHP_EMBED_TYPE="static"
```

Comment:
this existed twice so i removed one

Reply:
No, this is for macOS. It's two parts for different OS.